chrome/content/zotero/xpcom/attachments.js

```diff
@@ -35,6 +35,7 @@ Zotero.Attachments = new function(){
 	
 	var _findPDFQueue = [];
 	var _findPDFQueuePromise = null;
+	var queue;
```

Comment:
The only real change in that file that the [event handler](https://github.com/tnajdek/zotero/blob/bec33ee6d06e5ef6acb9d33e4fab882f386cbbfd/chrome/content/zotero/xpcom/attachments.js#L1506) is installed once for the queue, not on every run as it was previously. `queue` will be cleared correctly via closure, moving it out of the scope was unnecessary and I've fixed it.

Reply:
Sorry, if it was unclear, I just meant the name `queue` was too vague to be a top-level variable in this file. It seems like that's still present in the latest version?

Reply:
In this updated version `queue` is a function-scope variable in `addAvailablePDFs()` (as it was originally) declared [here](https://github.com/tnajdek/zotero/blob/progress-queue-table/chrome/content/zotero/xpcom/attachments.js#L1481) and assigned value [here](https://github.com/tnajdek/zotero/blob/bec33ee6d06e5ef6acb9d33e4fab882f386cbbfd/chrome/content/zotero/xpcom/attachments.js#L1509). 

Previously it was both declared and assigned value at the same time which is functionally the same because of hoisting. I've moved the definition up so that code is easier to read (I refer to `queue` in a callback [here](https://github.com/tnajdek/zotero/blob/bec33ee6d06e5ef6acb9d33e4fab882f386cbbfd/chrome/content/zotero/xpcom/attachments.js#L1506)).

If you think it's confusing I'm happy to change it the way it was, I only thought code would be more readable this way.

Reply:
OK, sorry, either GitHub was lying to me and showing me an old version or I was misreading. Looks good now.